.babelrc

```diff
@@ -9,5 +9,11 @@
         modules: false
       }
     ]
-  ]
+  ],
+  "env": {
+    "test": {
+      "plugins": [ "istanbul" ],
+      "sourceMaps": "inline"
+    }
+  }
```

Comment:
This is a nice little feature that babel supports. It allows for configuration to be applied optionally, depending on the value of the `BABEL_ENV` or `NODE_ENV` environment variable. So in the code above, inline source maps and the `istanbul` plugin are only only applied when `BABEL_ENV=test` or `NODE_ENV=test`.